prodtests/produce_QEDhits.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+# Script to produce hits coming from QED background interactions.
+# These hits will be overlayed on hits from normal events to produce
+# combined digit output.
+
+Usage()
+{
+  echo "Usage: ${0##*/} [-n Number of events ] [-e TGeant3|TGeant4]"
+  exit
+}
+
+engine="TGeant3"
+while [ $# -gt 0 ] ; do
+    case $1 in
+	-n) nev=$2;  shift 2 ;;
+	-e) engine=$2; shift 2 ;;
+	-h) Usage ;;
+	*) echo "Wrong input"; Usage;
+    esac
+done
+
+#---------------------------------------------------
+echo "Running simulation for $nev $collSyst events with $gener generator and engine $engine"
+#
+# we only need passive material and QED sensitive detectors close to beampipe
+#
+set -x
+o2-sim -j 8 -m PIPE ABSO DIPO SHIL ITS FT0 MFT -n"$nev" -e "$engine" --noemptyevents -g extgen --extGenFile ${O2_ROOT}/share/Generators/external/QEDLoader.C --configKeyValues "SimCutParams.maxRTracking=300" -o o2simqed
```

Comment:
@shahor02 : Could you check these settings? I guess we can constrain the tracking limits in R and Z. The sim seems quite fast.

Reply:
R cut should be irrelevant as the process is extremely low pT. Z will surely help. But 1st the FIT detectors should test the y_limits in the https://github.com/AliceO2Group/AliceO2/blob/f9b671e3b5169159458883ef4a5488d2544287ab/Generators/include/Generators/QEDGenParam.h#L27-L28 : start from wide coverage and shrink it until they see a difference in hits per orbit. 

The generation per even is fast because there are just 2 particles to track, but we need plenty of events due to the huge x-section. With the setting above it is 107kb (reported in the ``qedgenparam.ini`` written by generator) i.e. we need to inject ~14K QED events per 1 pbpb hadronic collisions. For the ITS I was generating in advance (in aliroot) 20k 1\mus slices (~17M events with settings above), enough for 1 TF. 
But we can generate a small fraction of these events and use them in a round robin.